util.py

```diff
@@ -102,3 +103,8 @@ def loadYaml(yaml_file):
 	with open(yaml_file) as fileobj:
 		return yaml.safe_load(fileobj)
 
+def getFqdn():
```

Comment:
Lets go one step further and have this function take the endpoint in as an argument, then return the fully qualified link.